Flabbergasted-Flamingos/bangazon-commerce-app
# Rb fixes

# Description
All the html's are prettified and looking sexy

## Number of Fixes
1

## Related Ticket(s)
a bunch

## Problem to Solve
The products are a nice card

## Proposed Changes
affects all current html's except register and login

## Expected Behavior
Behavious should stay the same

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb_fixes
pyman runserver
```
Have a play with eveything and make sure it works and makes sense
